distributed/minGPT-ddp/README.md

```diff
@@ -0,0 +1,150 @@
+# minGPT-DDP
```

Comment:
As a general note this whole PR doesn't quite match what we currently do in `examples`, as it feels more like a blog or tutorial. @hudeven @svekars what do you think should we change our policy?

Reply:
It is a tutorial, and not quite an "example" as the others in this repo. I initially had these on a personal fork, but we don't want to use personal repos. If including this requires changing/diluting the policy, perhaps we can identify another repository to house these? cc @malfet

Reply:
> I'm reusing [torchrecipes repo](https://github.com/facebookresearch/recipes) for Paved Path recipes and plan to make it flat structure as pytorch/examples. The main difference between the 2 repos would be recipes in torchrecipes are more end-to-end and flexible on dependencies, while examples are basic and mainly depend on pytorch. How about moving these tutorials to torchrecipes eventually?

Reply:
From the comment below, I think it makes sense to move this to torchrecipes eventually